uamqp/receiver.py

```diff
@@ -74,10 +74,10 @@ def _incoming_transfer(self, frame):
         if not self.received_delivery_id and not self._received_payload:
             pass  # TODO: delivery error
         if self._received_payload or frame[5]:  # more
-            self._received_payload += bytes(frame[11])  # TODO, is it possible to use memoryview directly for concatenation?
+            self._received_payload = bytes(bytearray(self._received_payload).append(bytearray(frame[11])))
         if not frame[5]:
             if self._received_payload:
-                message = decode_payload(memoryview(self._received_payload))  # TODO, is it possible to use memoryview directly for concatenation?
```

Comment:
[memoryview of bytes is immutable](https://stackoverflow.com/a/65854754/5307040)